### ISSUE TITLE
Add ngxs

### ISSUE THREAD

**Issue by splincode** [angular-checklist/angular-checklist#17]

**Comment by KwintenP** [comment]
Hi!
Thanks for opening this issue!

For some reason, I'm not really sure about adding this to the checklist. This is a library and I think we would like to keep this really focussed on angular core.
I know we added NGRX. And NGRX is also a library so it clashes with what I'm saying above. But for some reason, NGRX is kind of the de facto non official solution. So NGRX is kind of the exception here I would say.

Basically, what I'm saying is that I'm not sure NGXS should belong here as it is a library.

**Comment by KwintenP** [comment]
What do you think @d3lm.

**Comment by d3lm** [comment]
I would agree that NGRX is more or less an exception and that I would not create categories per library. While NGRX is not officially a first class citizen of the Angular repo, it's very commonly used. I'd like to stick to NGRX here. 

Maybe at some point we can think about making the checklist a little bit more sophisticated. For example having sub-categories. This would allow us to create a category such as `State Management` and have a sub-category for both NGRX and NGXS.

Just thinking out loud. But I don't want to make this too complex. It should stay simple because it was meant to be a tool that you can use to somewhat **quickly** check if you are on the right track or if there's anything you still need to work on. The more complexity we add to this app, the more time it takes to browse through the list. And the list is already naturally growing when items are being added.
